testing/_py/test_local.py

```diff
@@ -747,7 +747,7 @@ def test_setmtime(self):
             name = tempfile.mktemp()
             open(name, "w").close()
         try:
-            mtime = int(time.time()) - 100
+            mtime = int(_pytest.timing.perf_counter()) - 100
```

Comment:
We are dealing here with actual time-stamp, not a duration, so we should keep using `time`.

```suggestion
            mtime = int(_pytest.timing.time()) - 100
```